lib/uplinkc/object.go

```diff
@@ -320,7 +324,9 @@ func download_close(downloader C.DownloaderRef, cErr **C.char) {
 
 	err := download.rc.Close()
 	if err != nil {
-		*cErr = C.CString(fmt.Sprintf("%+v", err))
```

Comment:
Same question here. We weren't checking before, but now we are.

Reply:
(discussion above)